modules/oc-mirror-migration-process.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-migration-process_{context}"]
+= Migrating from oc-mirror plugin v1 to oc-mirror plugin v2
+
+Migrating from oc-mirror v1 to v2 requires manually updating the ImageSetConfiguration, adjusting mirroring commands, and ensuring proper cleanup of v1 artifacts. Follow these steps to transition smoothly:
```

Comment:
I like this section a lot because it is a straightforward list of steps to do in order to switch from using v1 to using v2.
my personal feeling is that the first module (differences) is a bit redundant... there are only a few information from the first module that are missing in this one, but that we could add here maybe.
Again, this is only my opinion... let's also let the others pitch in here.